easybuild/easyconfigs/p/Paraview/Paraview-4.2.0-intel-2014b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'Paraview'
+version = '4.2.0'
+
+homepage = "http://www.paraview.org"
+description = "Paraview is a scientific parallel visualizer"
+
+toolchain = {'name': 'intel', 'version': '2014b'}
+toolchainopts = {'usempi': False, 'pic': True}
+
+download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
+source_urls = ['http://www.paraview.org/paraview-downloads/%s' % download_suffix]
+sources = ["ParaView-v%(version)s-source.tar.gz"]
+
+patches = ['missingheader-%(version)s.patch']
+
+dependencies = [
+    ('Qt','4.8.6'),
+    ('Mesa', '10.4.5', '-Python-2.7.8'),
```

Comment:
wouldn't it be better go with 10.5.5 from #1618, and drop the Mesa-related stuff in this PR?

I'm not very keen on having lots of Mesa easyconfigs around, since they have a huge number of deps...